crates/noirc_evaluator/src/ssa/node.rs

```diff
@@ -787,7 +790,7 @@ impl Binary {
 
             BinaryOp::Udiv => {
                 if r_is_zero {
-                    todo!("Panic - division by zero");
+                    return Err(RuntimeErrorKind::Spanless(zero_div_error).into());
```

Comment:
Since we're constructing the same error in each spot, can we move the construction to the variable zero_div_error as well?

Reply:
Done